[2025-01-19T22:06:22.221+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-19T22:06:22.238+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DAG_AWS_PIPELINE.glue_sparkJob manual__2025-01-19T22:04:02.784376+00:00 [queued]>
[2025-01-19T22:06:22.409+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DAG_AWS_PIPELINE.glue_sparkJob manual__2025-01-19T22:04:02.784376+00:00 [queued]>
[2025-01-19T22:06:22.410+0000] {taskinstance.py:2866} INFO - Starting attempt 2 of 2
[2025-01-19T22:06:22.421+0000] {taskinstance.py:2889} INFO - Executing <Task(GlueJobOperator): glue_sparkJob> on 2025-01-19 22:04:02.784376+00:00
[2025-01-19T22:06:22.430+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=10033) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-19T22:06:22.431+0000] {standard_task_runner.py:72} INFO - Started process 10035 to run task
[2025-01-19T22:06:22.432+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'DAG_AWS_PIPELINE', 'glue_sparkJob', 'manual__2025-01-19T22:04:02.784376+00:00', '--job-id', '243', '--raw', '--subdir', 'DAGS_FOLDER/Ingesta_aws.py', '--cfg-path', '/tmp/tmp2k8ukzpt']
[2025-01-19T22:06:22.432+0000] {standard_task_runner.py:105} INFO - Job 243: Subtask glue_sparkJob
[2025-01-19T22:06:22.470+0000] {task_command.py:467} INFO - Running <TaskInstance: DAG_AWS_PIPELINE.glue_sparkJob manual__2025-01-19T22:04:02.784376+00:00 [running]> on host 59ff16f1d4fb
[2025-01-19T22:06:22.541+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Wilfrido Rondon' AIRFLOW_CTX_DAG_ID='DAG_AWS_PIPELINE' AIRFLOW_CTX_TASK_ID='glue_sparkJob' AIRFLOW_CTX_EXECUTION_DATE='2025-01-19T22:04:02.784376+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-19T22:04:02.784376+00:00'
[2025-01-19T22:06:22.543+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-19T22:06:22.543+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-19T22:06:22.543+0000] {logging_mixin.py:190} INFO - Current task name:glue_sparkJob state:running start_date:2025-01-19 22:06:22.238586+00:00
[2025-01-19T22:06:22.543+0000] {logging_mixin.py:190} INFO - Dag name:DAG_AWS_PIPELINE and current dag run status:running
[2025-01-19T22:06:22.543+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-19T22:06:22.544+0000] {glue.py:195} INFO - Initializing AWS Glue Job: aws_glue_default_job. Wait for completion: True
[2025-01-19T22:06:22.544+0000] {glue.py:369} INFO - Checking if job already exists: aws_glue_default_job
[2025-01-19T22:06:22.553+0000] {base.py:84} INFO - Retrieving connection 'aws_conn'
[2025-01-19T22:06:22.553+0000] {connection_wrapper.py:325} INFO - AWS Connection (conn_id='aws_conn', conn_type='aws') credentials retrieved from login and password.
[2025-01-19T22:06:23.734+0000] {glue.py:216} INFO - You can monitor this Glue Job run at: https://console.aws.amazon.com/gluestudio/home?region=us-east-2#/job/aws_glue_default_job/run/jr_4694814061ef32aece31e1637c275f12ecbf77b71d9647ec41033cad4f9ca43b
[2025-01-19T22:06:23.840+0000] {glue.py:352} INFO - Polling for AWS Glue Job aws_glue_default_job current run state with status RUNNING
[2025-01-19T22:06:29.995+0000] {glue.py:352} INFO - Polling for AWS Glue Job aws_glue_default_job current run state with status RUNNING
[2025-01-19T22:06:36.140+0000] {glue.py:352} INFO - Polling for AWS Glue Job aws_glue_default_job current run state with status RUNNING
[2025-01-19T22:06:42.270+0000] {glue.py:352} INFO - Polling for AWS Glue Job aws_glue_default_job current run state with status RUNNING
[2025-01-19T22:06:45.706+0000] {glue.py:352} INFO - Polling for AWS Glue Job aws_glue_default_job current run state with status RUNNING
[2025-01-19T22:06:51.810+0000] {glue.py:349} INFO - Exiting Job jr_4694814061ef32aece31e1637c275f12ecbf77b71d9647ec41033cad4f9ca43b Run State: FAILED
[2025-01-19T22:06:51.821+0000] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/amazon/aws/operators/glue.py", line 230, in execute
    glue_job_run = self.glue_job_hook.job_completion(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/amazon/aws/hooks/glue.py", line 300, in job_completion
    ret = self._handle_state(job_run_state, job_name, run_id, verbose, next_log_tokens)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/amazon/aws/hooks/glue.py", line 350, in _handle_state
    raise AirflowException(job_error_message)
airflow.exceptions.AirflowException: Exiting Job jr_4694814061ef32aece31e1637c275f12ecbf77b71d9647ec41033cad4f9ca43b Run State: FAILED
[2025-01-19T22:06:51.825+0000] {logging_mixin.py:190} INFO - Task instance in failure state
[2025-01-19T22:06:51.826+0000] {logging_mixin.py:190} INFO - Task start:2025-01-19 22:06:22.238586+00:00 end:2025-01-19 22:06:51.825260+00:00 duration:29.586674
[2025-01-19T22:06:51.826+0000] {logging_mixin.py:190} INFO - Task:<Task(GlueJobOperator): glue_sparkJob> dag:<DAG: DAG_AWS_PIPELINE> dagrun:<DagRun DAG_AWS_PIPELINE @ 2025-01-19 22:04:02.784376+00:00: manual__2025-01-19T22:04:02.784376+00:00, state:running, queued_at: 2025-01-19 22:04:02.792807+00:00. externally triggered: True>
[2025-01-19T22:06:51.826+0000] {logging_mixin.py:190} INFO - Failure caused by Exiting Job jr_4694814061ef32aece31e1637c275f12ecbf77b71d9647ec41033cad4f9ca43b Run State: FAILED
[2025-01-19T22:06:51.826+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=DAG_AWS_PIPELINE, task_id=glue_sparkJob, run_id=manual__2025-01-19T22:04:02.784376+00:00, execution_date=20250119T220402, start_date=20250119T220622, end_date=20250119T220651
[2025-01-19T22:06:51.850+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-19T22:06:51.851+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 243 for task glue_sparkJob (Exiting Job jr_4694814061ef32aece31e1637c275f12ecbf77b71d9647ec41033cad4f9ca43b Run State: FAILED; 10035)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3005, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3159, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3183, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 422, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/amazon/aws/operators/glue.py", line 230, in execute
    glue_job_run = self.glue_job_hook.job_completion(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/amazon/aws/hooks/glue.py", line 300, in job_completion
    ret = self._handle_state(job_run_state, job_name, run_id, verbose, next_log_tokens)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/amazon/aws/hooks/glue.py", line 350, in _handle_state
    raise AirflowException(job_error_message)
airflow.exceptions.AirflowException: Exiting Job jr_4694814061ef32aece31e1637c275f12ecbf77b71d9647ec41033cad4f9ca43b Run State: FAILED
[2025-01-19T22:06:51.904+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2025-01-19T22:06:51.928+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-19T22:06:51.934+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
